Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml

```diff
@@ -61,7 +61,7 @@
                             <li><h3>#{msgs.dashboard}<h:outputText rendered="#{LoginForm.maximaleBerechtigung == 1}"> <i>(Admin)</i></h:outputText></h3></li>
                             <li class="nav-pic-text"><h:link value="#{msgs.schritte}" outcome="tasks"><i class="fa fa-bell fa-lg"/></h:link></li>
                             <li class="nav-pic-text"><h:link value="#{msgs.prozesse}" outcome="processes"><i class="fa fa-hourglass-3 fa-lg"/></h:link></li>
-                            <li class="nav-pic-text"><h:link value="#{msgs.projekte}" outcome="projects" onclick="#{ProjekteForm.setLocked(true)}"><i class="fa fa-folder-open fa-lg"/></h:link></li>
+                            <li class="nav-pic-text"><h:link value="#{msgs.projekte}" outcome="projects"><i class="fa fa-folder-open fa-lg"/></h:link></li>
```

Comment:
Is this change really wanted? It is introduced by @Beacze .

Reply:
Without it I couldn't save copied project. I don't know for what is used this parameter.

Reply:
This onclick action has to remain in the code, probably. It was introduced in the "accidental change prevention" that @IkramMaalej introduced in #1237. Instead, the lock should be set to `false` when someone clicks the `duplicateProject` button. I will add a corresponding commit.

Reply:
Is this now solved?

Reply:
Yes, this is resolved (after discussing the issue with @IkramMaalej, we decided that locking the project shouldn't be done in the header but instead only when someone clicks on the "project edit" button in the list view, which has been added in commit 9d73a74d7047899b706f9460ca5a066903c1d1d7)